packages/twenty-server/src/engine/seeder/seeder.service.ts

```diff
@@ -64,98 +65,155 @@ export class SeederService {
 
     const entityManager = workspaceDataSource.createEntityManager();
 
-    const filteredFields = metadataSeeds.fields.filter((field) =>
-      objectMetadataAfterFieldCreation.fields.some(
-        (f) => f.name === field.name || f.name === `name`,
-      ),
+    const filteredFieldMetadataSeeds = objectMetadataSeed.fields.filter(
+      (field) =>
+        objectMetadataAfterFieldCreation.fields.some(
+          (f) => f.name === field.name || f.name === `name`,
+        ),
     );
 
-    if (filteredFields.length === 0) {
-      throw new Error('No fields found for seeding, check metadata');
+    if (filteredFieldMetadataSeeds.length === 0) {
+      throw new Error('No fields found for seeding, check metadata file');
     }
 
-    filteredFields.unshift({
-      name: 'name',
-      type: FieldMetadataType.TEXT,
-      label: 'Name',
-    });
+    this.addAutomaticalyCreatedNameFieldToFieldMetadataSeeds(
```

Comment:
I'm not sure Automatically is necessary here?

Reply:
How should explain the fact that createOne is automatically adding a name field that we don't have in our metadata seeds and that we cannot control because it's embedded in createOne behavior so we have to add it to our metadata seeds afterwards to account for that automatic behavior ?

Reply:
I just kept addNameFieldToFieldMetadataSeeds is it ok ?